daemon/src/model.rs

```diff
@@ -618,6 +618,10 @@ impl FundingRate {
     pub fn to_decimal(&self) -> Decimal {
         self.0
     }
+
+    pub fn short_pays_long(&self) -> bool {
+        self.0.is_sign_negative()
```

Comment:
This will be `false` on 0, not sure what the implication is!

Reply:
I am not sure what to do with the zero case - for the zero case the fee should be zero. It is irrelevant what the fee flow then is. I did not depict it here. If you want I can also use the fee-flow instead of just returning `bool`?

Reply:
`FeeFlow` now holds an `Amount` so not ideal, I think we can leave it for now!

Perhaps add a test that covers a rate of 0?